helper/schema/schema.go

```diff
@@ -131,6 +131,9 @@ type Schema struct {
 	// This string is the message shown to the user with instructions on
 	// what do to about the removed attribute.
 	Removed string
+
+	// ValidateFunc allows individual fields to validate
```

Comment:
Can we add that the interface{} arg is guaranteed to be the schema type? I'm assuming that's the case: we already validated the type is correct.